### PR TITLE
Add a footnote to explain potential confusion

### PR DESCRIPTION
**Pull request by ra2dd**
In the following tutorial user is tasked to add 3 to 74 (which type is string). First we add 3 to string so the result is 743. Tutorial explains that we need to change data type in the operation to get desirable result. After changing type in calculation output is 746, not 77 because we added 3 to 743. This may confuse the user to think that he made something wrong(also 743 and 746 are very similar visually, user may not spot the difference) in tutorial we wanted to get 77.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/math/index.md [hunk 1]
```diff
@@ -121,6 +121,8 @@ To fix the calculation, you can do this:
 Number(myNumber) + 3;
```
Comment (teoli2003) [inline files/en-us/learn/javascript/first_steps/math/index.md:121]: I would rather re-start from scratch the example:
```suggestion
let myNumber = '74';
Number(myNumber) + 3;
```

File: files/en-us/learn/javascript/first_steps/math/index.md [hunk 2]
```diff
@@ -121,6 +121,8 @@ To fix the calculation, you can do this:
...
 ```
 
+Note that the result is 746. Our script took the string we modified - 743, changed temporarily its type to number and added 3.
```
Comment (teoli2003) [inline files/en-us/learn/javascript/first_steps/math/index.md:124]:
```suggestion
The result is then 77, as initially expected.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I think we should restart the example rather than continue the example. It is clearer.

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**